src/api/app/models/bs_request_action_change_devel.rb

```diff
@@ -30,6 +30,10 @@ def uniq_key
     "changedevel/#{target_project}/#{target_package}"
   end
 
+  def name
```

Comment:
In the method above you see what makes this action uniq. It's also how we should identify this action in the dropdown. Same for all the other action names.

Reply:
Okay, we are now using the `uniq_key` but in a modified version that humanizes the keys